sphinxcontrib/confluencebuilder/storage/translator.py

```diff
@@ -1746,6 +1746,20 @@ def visit_confluence_expand(self, node):
     def depart_confluence_expand(self, node):
         self.body.append(self.context.pop())  # macro
 
+    def visit_confluence_excerpt(self, node):
+        self.body.append(self._start_ac_macro(node, 'excerpt'))
+        if 'name' in node:
+            self.body.append(
+                self._build_ac_param(node, 'name', node['name']))
+        self.body.append(
+            self._build_ac_param(node, 'atlassian-macro-output-type', "INLINE"))
```

Comment:
From what I can tell, `atlassian-macro-output-type` may also accept a `block` value. Maybe both option types should be supported through a directive definition (I haven't had time to check what the difference is)?